tests/rptest/clients/rpk.py

```diff
@@ -1426,6 +1454,80 @@ def acl_create_allow_cluster(self,
 
         return output
 
+    def acl_create(self, acl: RPKACLInput):
+
+        cmd = [
+            self._rpk_binary(),
+            "security",
+            "acl",
+            "create",
+        ] + self._schema_registry_conn_settings() + self._kafka_conn_settings(
+        )
+
+        def append_flag(flag: str, values: list[str]):
+            if values:
+                cmd.extend([flag, ",".join(values)])
+
+        def append_bool_flag(flag: str, value: bool):
+            if value:
+                cmd.append(flag)
+
+        append_flag("--allow-principal", acl.allow_principal)
+        append_flag("--deny-principal", acl.deny_principal)
+        append_flag("--allow-role", acl.allow_role)
+        append_flag("--deny-role", acl.deny_role)
+        append_flag("--allow-host", acl.allow_host)
+        append_flag("--deny-host", acl.deny_host)
+        append_flag("--topic", acl.topic)
+        append_flag("--group", acl.group)
+        append_flag("--operation", acl.operation)
+        append_flag("--transactional-id", acl.txn_id)
+        append_bool_flag("--cluster", acl.cluster)
+        append_flag("--registry-subject", acl.registry_subject)
+        append_bool_flag("--registry-global", acl.registry_global)
+
+        if acl.resource_pattern_type:
+            cmd += ["--resource-pattern-type", acl.resource_pattern_type]
+
+        return self._execute(cmd)
+
+    def acl_delete(self, acl: RPKACLInput):
```

Comment:
would be great to have `list` test too, based you dont' use `--no-confirm` on delete acl test

Reply:
we do have one, the list command was already created in the rpk client. I use it a lot during the test (listing with filter, subsystem and all)